include/podio/Frame.h

```diff
@@ -150,7 +150,8 @@ class Frame {
   ///
   /// @tparam FrameDataT Arbitrary data container that provides access to the
   ///                    collection buffers as well as the metadata, when
-  ///                    requested by the Frame.
+  ///                    requested by the Frame. The unique_ptr has to be checked
+  ///                    for validity or this constructor will throw an std::invalid_argument
```

Comment:
```suggestion
  ///                    requested by the Frame. The unique_ptr has to be checked
  ///                    for validity before calling this construtor.
  ///
  /// @throws std::invalid_argument if the passed pointer is a nullptr.
```

since doxygen has this tag.